policyengine_us/variables/gov/states/va/tax/income/deductions/itemized_deductions/va_itemized_deductions_head.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class va_itemized_deductions_head(Variable):
```

Comment:
@MaxGhenis are there any additional steps we need to take to prorate the itemized deductions?